assets/src/application/ui/display/icons/svgs/withEnhance.tsx

```diff
@@ -23,10 +24,14 @@ const withEnhance = <P extends IconProps>(WrappedComponent: React.ComponentType<
 			props.className
 		);
 
-		return <WrappedComponent {...(props as P)} className={className} />;
+		return <WrappedComponent {...(props as P)} className={className} ref={forwardedRef} />;
 	};
 
-	return WithEnhance;
+	const ForwardedComponentWithEnhance = (props: P, ref: Ref) => {
+		return <WithEnhance {...props} forwardedRef={ref} />;
+	};
```

Comment:
this icon was missing tooltip: 


![image](https://user-images.githubusercontent.com/1477453/82573683-c15d4d00-9b8e-11ea-900f-429c44e498ec.png)

Reply:
ahhhh GREAT catch 💯